src/data/queries/vetCenterHealthServices.ts

```diff
@@ -31,9 +33,11 @@ export const formatter: QueryFormatter<
       commonlyTreatedCondition:
         serviceEntity.field_commonly_treated_condition || null,
       vetCenterServiceDescription:
-        serviceEntity.field_vet_center_service_descrip || null,
-      description: serviceEntity.description?.processed || null,
-      body: item.field_body?.processed || null,
+        getHtmlFromDrupalContent(
+          serviceEntity.field_vet_center_service_descrip
+        ) || null,
+      description: getHtmlFromField(serviceEntity.description) || null,
```

Comment:
This `description` field does not appear to be used anywhere. This query formatter is specific to the Vets Center. I'd like to know from someone familiar with the Drupal side of things if these fields' value is coming from the same source (`description` vs `field_vet_center_service_descrip `).

Reply:
sorry, I take that back. I see this is an included taxonomy term that does have a description value. It is different data than what is in `field_vet_center_service_descrip` and also its an object, so I think it should be `getHtmlFromField(serviceEntity.description?.processed) || null`.

Reply:
oh, I see its not used in the render...yeah lets kill it then.